recipe/meta.yaml

```diff
@@ -90,6 +90,9 @@ requirements:
     - {{ cdt('libxxf86vm') }}            # [linux]
     - {{ cdt('libxrandr') }}             # [linux]
     - {{ cdt('alsa-lib') }}              # [linux]
+
+    - python                                # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}    # [build_platform != target_platform]
```

Comment:
Also need:
```
- sip                                    # [build_platform != target_platform]
```

Reply:
`pyqtwebkit` also needs this:
```
- pyqt-builder                           # [build_platform != target_platform]
```
Might be worth a try?

Reply:
With that if platform != ... Right?

Reply:
Yep - that's right only for cross compilation. No idea what all this stuff does, but hopefully if we copy enough of the `pyqtwebkit` recipe we'll get it working....

Reply:
Last one was different error, so you are getting there. I have no idea of what I am doing :-p